TodoTests/XTests.swift

```diff
@@ -11,10 +11,16 @@ import XCTest
 @testable import Todo //DONT FORGET
 
 
+
 class XTests: XCTestCase {
     
-    func testThatSomething() {
+    let td = TodoViewModel()
+    
+    func testTransformedData() {
+        
+        let s = td.transformedDatasource("Jerry")
         
+        XCTAssert(s[0] == "Jerry:")
```

Comment:
I made the issue: https://github.com/alonecuzzo/Todo/issues/2 so we can discuss a possible approach to allowing the view model to deal with inputted data.